_posts/2024-05-06-message-property-injection.markdown

```diff
@@ -0,0 +1,124 @@
+---
+# Layout
+layout: post
+title:  "Add IncomingMessageProperty injection support anf Kafka transport enhancements in CoreWCF"
+date:   2024-05-06 13:00:00 +0200
```

Comment:
Update the date to something more current.

Reply:
Updated!